.ci/ci_job_flags.sh

```diff
@@ -96,21 +96,21 @@ case "${CI_JOB}" in
 	export KATA_HYPERVISOR="qemu"
 	export KUBERNETES="no"
 	;;
-"CRI_CONTAINERD"|"CRI_CONTAINERD_K8S"|"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD")
+"CRI_CONTAINERD"|"CRI_CONTAINERD_K8S"|"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
 	# This job only tests containerd + k8s
 	init_ci_flags
 	export CRI_CONTAINERD="yes"
 	export CRI_RUNTIME="containerd"
 	export KATA_HYPERVISOR="qemu"
 	case "${CI_JOB}" in
-		"CRI_CONTAINERD_K8S")
+		"CRI_CONTAINERD_K8S" | "CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
 			export KUBERNETES="yes"
 			;;
-		"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD")
+		"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD" | "CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
 			# Export any CC specific environment variables
 			export CCV0="yes"
 			export UMOCI=yes
-			if [ "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD" ]; then
+			if [ "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD" || "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD_K8S" ]; then
 				export SKOPEO=yes
 			fi
```

Comment:
You might want to consider changing
```
if [ "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD" || "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD_K8S" ]; then
	export SKOPEO=yes
fi
```
to
```
[[ "${CI_JOB}" =~ SKOPEO ]] && export SKOPEO=yes
```
as that matches the approach I've seen used elsewhere in the code base